ts/sagas/wallet.ts

```diff
@@ -863,7 +854,8 @@ export function* watchWalletSaga(pagoPaClient: PagoPaClient): Iterator<Effect> {
     if (action.type === ADD_CREDIT_CARD_REQUEST) {
       yield fork(
```

Comment:
@fgiobergia btw why is this a fork instead of a call?

Reply:
@fgiobergia Yes indeed, but what other request you would handle before handling the result of saving a card? I believe the app will actually wait for the card to be saved to present a feedback to the user?

I mean, by forking that saga, the user could save another card and while he's doing that he could receive an errore be cose the first card he was trying to save has failed.